src/common/colorspaces.c

```diff
@@ -246,49 +248,44 @@ static cmsHPROFILE dt_colorspaces_create_lab_profile()
 }
 
 static cmsHPROFILE _create_lcms_profile(const char *desc, const char *dmdd,
-                                        const cmsCIEXYZTRIPLE *primaries, const cmsToneCurve *trc, gboolean v2)
+                                        const cmsCIExyY *whitepoint, const cmsCIExyYTRIPLE *primaries, cmsToneCurve *trc,
+                                        gboolean v2)
 {
-  cmsCIEXYZ black = { 0, 0, 0 };
+  cmsHPROFILE profile;
+  cmsToneCurve *out_curves[3];
+  cmsMLU *mlu1 = cmsMLUalloc(NULL, 1);
+  cmsMLU *mlu2 = cmsMLUalloc(NULL, 1);
+  cmsMLU *mlu3 = cmsMLUalloc(NULL, 1);
+  cmsMLU *mlu4 = cmsMLUalloc(NULL, 1);
 
-  cmsHPROFILE profile = cmsCreateProfilePlaceholder(0);
+  out_curves[0] = out_curves[1] = out_curves[2] = trc;
 
-  if(!profile) return NULL;
+  profile = cmsCreateRGBProfile(whitepoint, primaries, out_curves);
 
   if(v2)
     cmsSetProfileVersion(profile, 2.1);
 
-  cmsMLU *mlu0 = cmsMLUalloc(NULL, 1);
-  cmsMLUsetASCII(mlu0, "en", "US", "Public Domain");
-  cmsMLU *mlu1 = cmsMLUalloc(NULL, 1);
-  cmsMLUsetASCII(mlu1, "en", "US", desc);
-  cmsMLU *mlu2 = cmsMLUalloc(NULL, 1);
-  cmsMLUsetASCII(mlu2, "en", "US", "Darktable");
-  cmsMLU *mlu3 = cmsMLUalloc(NULL, 1);
+  cmsSetHeaderFlags(profile, cmsEmbeddedProfileTrue | cmsUseAnywhere);
+  cmsSetHeaderRenderingIntent(profile, INTENT_RELATIVE_COLORIMETRIC);
```

Comment:
What you would suggest? perceptual?

Reply:
Ideally, nothing. I don't know if LCMS2 can just skip the intent and keep off-gamut values as-is, but it would be the sensible thing to do here.

Reply:
It can, there was no intent in previous code in fact. Now my thinking is that does nothing on the creation of the colorspace, or does it? The intent would only be used while mapping an bunch of pixels in the colorspace. But anyway, I'll remove this.